src/main/resources/application.properties

```diff
@@ -0,0 +1,24 @@
+#spring.http.converters.preferred-json-mapper=gson
+
+spring.jackson.serialization.FAIL_ON_EMPTY_BEANS=false
+
+### default connection pool
+#spring.datasource.hikari.connectionTimeout=20000
+#spring.datasource.hikari.maximumPoolSize=5
+
+#jpa properties
+spring.jpa.hibernate.ddl-auto=update
+spring.jpa.show-sql=true
+spring.jpa.generate-ddl=true
+spring.jpa.database=POSTGRESQL
+spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
+
+## PostgreSQL
+spring.datasource.platform=postgres
+spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
+spring.datasource.username=postgres
+spring.datasource.password=qwertyui
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.PostgreSQLDialect
```

Comment:
I think we should probably remove the username and password since this is now public in git. Both can be configured with environment variables instead

Reply:
In eclipse, go to Run > Run Configurations > Environment  and then add environment variables. They can then be called in applications.properties using ${DBName}, etc.

Reply:
I am doing this as part of the ci setup.